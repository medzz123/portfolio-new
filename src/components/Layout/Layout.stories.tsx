import { Meta, Story } from '@storybook/react/types-6-0';
import React from 'react';

import Layout from './Layout';

export default {
  title: 'Components/Layout',
  component: Layout,
} as Meta;

const Template: Story = (storyArguments) => <Layout {...storyArguments} />;

export const Default = Template.bind({});
