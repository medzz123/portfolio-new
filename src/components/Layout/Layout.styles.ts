import { styled } from '@theme/config';

export const LayoutContainer = styled('div', {
  maxWidth: '$6xl',
  margin: '0 auto',
  padding: '0 $24',
});
