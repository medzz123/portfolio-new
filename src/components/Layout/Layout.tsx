import React, { FunctionComponent } from 'react';

import { LayoutContainer } from './Layout.styles';

const Layout: FunctionComponent = (props) => {
  const { children } = props;

  return <LayoutContainer data-testid="layout">{children}</LayoutContainer>;
};

export default Layout;
