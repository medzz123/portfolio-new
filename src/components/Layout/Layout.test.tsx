import { render } from '@test/library';
import React from 'react';

import Layout from './Layout';

test('Layout Test', () => {
  const { container } = render(<Layout>Hello</Layout>);

  expect(container).toMatchSnapshot();
});
