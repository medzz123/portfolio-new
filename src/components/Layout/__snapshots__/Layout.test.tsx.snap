// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Layout Test 1`] = `
<div>
  <div
    class="sxsw5wg"
    data-testid="layout"
  >
    Hello
  </div>
</div>
`;
